.gitignore

```diff
@@ -17,14 +17,8 @@ TreeSheets/\.vs/treesheets/v15/
 build/treesheets/language/
 
 TS/scripts/\.con\.log
-TS/treesheets
-TS/2
 
-CMakeCache.txt
```

Comment:
why are these removed from `.gitignore` ? people may still default to in-tree builds where this is useful

Reply:
I do not see any value in in-tree builds but re-added that.